lib/top_hangman/round.rb

```diff
@@ -1,9 +1,23 @@
+require_relative 'render'
+
 module TopHangman
   class Round
     attr_reader :word
 
     def initialize(word)
       @word = word
+      @end_of_round = false
+      @guesses = []
+      @guess = nil
```

Comment:
I think renaming this to `@current_guess` helps making it more self-explanatory

Reply:
`@current_guess` instead of `@guess` ? I like it